model/host/db.go

```diff
@@ -278,7 +277,7 @@ var IsActive = db.Query(
 		StartedByKey: evergreen.User,
 		StatusKey: bson.M{
 			"$nin": []string{
-				evergreen.HostTerminated, evergreen.HostDecommissioned, evergreen.HostInitializing,
+				evergreen.HostTerminated, evergreen.HostDecommissioned,
```

Comment:
I think you want to add provisioning to this otherwise we'd end up calling SpawnHost more than once...

Reply:
I think better to make it safe since its not in test code (though we could move it there.

Reply:
done